File: MainApp/src/main/java/com/sleticalboy/dailywork/bt/ble/BleException.java
package com.sleticalboy.dailywork.bt.ble;

import android.util.AndroidException;

/**
 * Created on 20-8-19.
 *
 * @author Ben binli@grandstream.cn
 */
public class BleException extends AndroidException {

    public BleException(String msg) {
        super(msg);
    }

    public BleException(String msg, Throwable cause) {
        super(msg, cause);
    }
}

File: MainApp/src/main/java/com/sleticalboy/learning/bt/ble/BleException.kt
package com.binlee.learning.bt.ble

import android.util.AndroidException

/**
 * Created on 20-8-19.
 *
 * @author Ben binli@grandstream.cn
 */
class BleException : AndroidException {

  constructor(msg: String?) : super(msg)

  constructor(msg: String?, cause: Throwable?) : super(msg, cause)
}